Resources/Linux/io.github.elasota.aerofoil.releases.xml

```diff
@@ -0,0 +1,10 @@
+<releases>
+  <release version="1.1.2" date="2021-11-10">
+    <url>https://github.com/elasota/Aerofoil/releases/tag/1.1.2</url>
+        <description>
```

Comment:
What's this information for?  Is this for the user machine, or for app listings somewhere?  (Or both?)

Reason I ask is the current (unreleased) version number is 1.1.6 (see https://github.com/elasota/Aerofoil/blob/master/GpCommon/GpBuildVersion.h), and the 1.1.2 release tag doesn't have a Linux build.

Reply:
The releases section is entirely informational, for display on e.g. the Flathub listing or in graphical package managers. For an example, have a look at the Flathub listing for [AMC Squad](https://flathub.org/apps/io.itch.amcsquad.amcsquad) and the [releases.xml](https://gitlab.com/fpiesche/amc-squad-data/-/blob/main/meta/io.itch.amcsquad.amcsquad.releases.xml) for that package.

At this point the initial Flathub distribution for Aerofoil would be a development snapshot based on a pinned commit ID; in the future Flathub's infrastructure would automatically update the build as new version tags are created in the git repo here. I could add a release entry specifically for the first Flathub build with a changelog based on what commit I build that from, if you think that makes sense?